4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4650 @@
+{
```

Comment:
<p>I like this new example, but the connection to regularization and ML should be explicit too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Got it, will try and reincorporate